src/lib.rs

```diff
@@ -322,13 +322,46 @@ where
 
 /// Run the given function under a randomized concurrency scheduler for some number of iterations.
 /// Each iteration will run a (potentially) different randomized schedule.
+/// When the environment variable SHUTTLE_REPLAY_SEED is set to a u64, this number will be used
+/// as the seed to initialize the random scheduler.
 pub fn check_random<F>(f: F, iterations: usize)
 where
     F: Fn() + Send + Sync + 'static,
 {
     use crate::scheduler::RandomScheduler;
 
-    let scheduler = RandomScheduler::new(iterations);
+    let seed_env = std::env::var("SHUTTLE_REPLAY_SEED");
```

Comment:
Not sure how much I care, but calling it `SHUTTLE_REPLAY_SEED` is odd, as it strictly speaking is independent of replaying.

Reply:
maybe `SHUTTLE_RANDOM_SEED`?

Reply:
Yeah, given the changes, let's go with `SHUTTLE_RANDOM_SEED`